impacket/krb5/kerberosv5.py

```diff
@@ -92,7 +92,7 @@ def sendReceive(data, host, kdcHost, port=88):
 
     return r
 
-def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcHost=None, requestPAC=True, serverName=None):
+def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcHost=None, requestPAC=True, serverName=None, kerberoast_no_preauth=False):
```

Comment:
aside from GetUsersSPN and getTGT calling this function with _service_ parameter. it seems that inside this function service is not being used at all, so, what would be the reason of passing this parameter?

Reply:
`service` was a mistake, the parameter name was switched to `serverName` not too long ago, and `serverName` is used in multiple places in the function, I'm not sure I understand the question.